src/cmdline/mod.rs

```diff
@@ -39,6 +40,8 @@ impl fmt::Display for Error {
     }
 }
 
+impl StdError for Error {}
```

Comment:
Is this really the conventional way to do it rather than:

`impl std::error::Error for Error {}`

Reply:
Just to keep same style with other implementations in linux-loader. Actually I prefer:
impl std::error::Error for Error {}

Reply:
If the rest of the code does it the you should continue to be consistent.

Reply:
I also prefer impl std::error::Error; It is more clear that StdError :(

Reply:
let's do it:)